docker/nginx/nginx.conf

```diff
@@ -35,8 +35,8 @@ server {
 
     root /srv/payments-ui;
 
-    location /payment-card/ {
-        alias /srv/payments-ui/;
```

Comment:
I was confused when I first saw this path stuff so it might be good to add a comment about how it's not necessary for production usage but nice to have for development.